Fix build warning in usnic

This may be bogus, but please check:

  CC     prov/usnic/src/usnic_direct/libnl_utils_common.lo
prov/usnic/src/usnic_direct/libnl_utils_common.c: In function \u2018usnic_nl_rt_lookup\u2019:
prov/usnic/src/usnic_direct/libnl_utils_common.c:248: warning: \u2018unlsk\u2019 may be used uninitialized in this function

--- a/prov/usnic/src/usnic_direct/libnl_utils_common.c
+++ b/prov/usnic/src/usnic_direct/libnl_utils_common.c
@@ -251,6 +251,7 @@ int usnic_nl_rt_lookup(uint32_t src_addr, uint32_t dst_addr, int oif,
 	struct usnic_rt_cb_arg	arg;
 	int			err;
 
+	unlsk = NULL;
 	err = usnic_nl_sk_alloc(&unlsk, NETLINK_ROUTE);
 	if (err)
 		return err;

